kafka/ci/resources/docker-compose-single-broker.yml

```diff
@@ -0,0 +1,20 @@
+version: '2'
+services:
+  zookeeper:
+    image: wurstmeister/zookeeper
+    ports:
+      - "2181:2181"
+  kafka:
+    image: "wurstmeister/kafka:${FLAVOR_VERSION}"
+    ports:
+      - "${EXTERNAL_PORT}:9092"
+      - "${EXTERNAL_JMX_PORT}:9999"
+    environment:
+      # KAFKA_ADVERTISED_HOST_NAME: 192.168.99.100
```

Comment:
is this needed or can we clean it up?
edit: oh is it needed for the tests that don't work yet?